backend/dataall/db/api/target_type.py

```diff
@@ -37,6 +37,5 @@ def is_supported_target_type(target_type):
 
 
 TargetType("environment", permissions.GET_ENVIRONMENT, permissions.UPDATE_ENVIRONMENT)
-TargetType("mlstudio", permissions.GET_SGMSTUDIO_NOTEBOOK, permissions.UPDATE_SGMSTUDIO_NOTEBOOK)
```

Comment:
Why did we delete this line? It should be re-added in the module initialization

Reply:
Isn't it here (`backend/dataall/modules/mlstudio/__init__.py:21`) ?